docs/get_started.md

```diff
@@ -28,51 +28,75 @@
+   
+
+#### <a href="tutorials.html"><ins>Interactive Tutorials - Jupyter Notebooks</ins></a>
+Start with interactive Python tutorials that show the basics of model inferencing, the OpenVINO API, how to convert models to OpenVINO format, and more.
+* <a href="001-hello-world-with-output.html">Hello Image Classification</a> - Load an image classification model in OpenVINO and use it to apply a label to an image
+* <a href="002-openvino-api-with-output.html">OpenVINO Runtime API Tutorial</a> - Learn the basic Python API for working with models in OpenVINO
+* <a href="101-tensorflow-to-openvino-with-output.html">Convert TensorFlow Models to OpenVINO</a>
+* <a href="102-pytorch-onnx-to-openvino-with-output.html">Convert PyTorch Models to OpenVINO</a>
+
+#### <a href="openvino_docs_OV_UG_Samples_Overview.html"><ins>OpenVINO Code Samples</ins></a>
+View sample code for various C++ and Python applications that can be used as a starting point for your own application. For C++ developers, step through the <a href="openvino_docs_get_started_get_started_demos.html">Basic OpenVINO Workflow</a> to learn how to build and run an image classification program that uses OpenVINO’s C++ API.
+      
+#### <a href="openvino_docs_OV_UG_Integrate_OV_with_your_application.html"><ins>Integrate OpenVINO With Your Application</ins></a>
+Learn how to use the OpenVINO API to implement an inference pipeline in your application.
+
+
+### <a name="openvino-advanced-features"></a>OpenVINO Advanced Features
+OpenVINO provides features to improve your model’s performance, optimize your runtime, maximize your application’s throughput on target hardware, and much more. Visit the links below to learn more about these features and how to use them.
+
+#### Model Compression and Quantization
+Use OpenVINO’s model compression tools to reduce your model’s latency and memory footprint while maintaining good accuracy.
+* Tutorial - <a href="111-detection-quantization-with-output.html">OpenVINO Post-Training Model Quantization</a>
+* Tutorial - <a href="305-tensorflow-quantization-aware-training-with-output.html">Quantization-Aware Training in TensorFlow with OpenVINO NNCF</a>
+* Tutorial - <a href="302-pytorch-quantization-aware-training-with-output.html">Quantization-Aware Training in PyTorch with NNCF</a>
+* <a href="openvino_docs_model_optimization_guide.html">Model Optimization Guide</a>
+
+#### Automated Device Configuration
+OpenVINO’s hardware device configuration features enable you to write an application once and deploy it anywhere with optimal performance.
+* Increase application portability with <a href="openvino_docs_OV_UG_supported_plugins_AUTO.html">Automatic Device Selection (AUTO)</a>
```

Comment:
If this is written in md, I suppose the references should be in md format (except for notebooks because notebooks are in another repo). Should we change the links to other pages to md format? @kblaszczak-intel

Reply:
@EdjeElectronics could you help change the reference links to Markdown format?

Reply:
Okay, I can do that! I'll go through and change the rest of them. Once I'm done, please look through the new reference links to make sure I implemented them correctly.

Reply:
Yes, referring to source files is much better. We can use the typical markdown notation but also the (@ref article_label) style, which is a bit safer because does not rely on the file path (it is more reliable for content from different sources, like notebooks.